packages/kit/src/runtime/client/fetcher.js

```diff
@@ -36,10 +41,14 @@ if (DEV) {
 		const cutoff = stack_array.findIndex((a) => a.includes('load@') || a.includes('at load'));
 		const stack = stack_array.slice(0, cutoff + 2).join('\n');
 
-		const heuristic = can_inspect_stack_trace
+		const inLoadHeuristic = can_inspect_stack_trace
```

Comment:
we use `_` names for internal code
```suggestion
		const in_load_heuristic = can_inspect_stack_trace
```